components/screens/BlogScreen/BlogScreen.js

```diff
@@ -1,62 +1,98 @@
 import { useIsFocused } from '@react-navigation/native';
 import React from 'react';
 import { useState, useEffect } from 'react';
-import { View, Text, ScrollView, Button } from 'react-native';
+import { View, Text, ScrollView, Button, Alert, ActivityIndicator } from 'react-native';
 import { Image } from 'react-native-elements';
 import InAppBrowser from 'react-native-inappbrowser-reborn';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import { Styles } from './BlogStyles';
 
+
+async function openLink(urlToOpen) {
+  try {
+    if (await InAppBrowser.isAvailable()) {
+      const result = await InAppBrowser.open(urlToOpen, {
+        showTitle: false,
+        toolbarColor: '#0176ae',
+        secondaryToolbarColor: 'black',
+        navigationBarColor: 'black',
+        navigationBarDividerColor: 'white',
+        enableUrlBarHiding: true,
+        enableDefaultShare: true,
+        forceCloseOnRedirection: false,
+        hasBackButton: true,
+      })
+    }
+    else Linking.openURL(urlToOpen)
+  } catch (error) {
+    console.log(error.message)
+  }
```

Comment:
Just want to say I love this! Looks very slick and professional.

I think once we get all our screens pushed to main we should implement this or something similar for all our InAppBrowsers!

Reply:
Thank you Caleb!